src/libraries/System.IO.Pipes.AccessControl/src/System.IO.Pipes.AccessControl.csproj

```diff
@@ -1,20 +1,26 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AssemblyName>System.IO.Pipes.AccessControl</AssemblyName>
-    <IncludeDefaultReferences>false</IncludeDefaultReferences>
     <IsPartialFacadeAssembly Condition="'$(TargetsWindows)' == 'true'">true</IsPartialFacadeAssembly>
     <OmitResources Condition="'$(TargetsWindows)' == 'true'">true</OmitResources>
     <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetsWindows)' != 'true'">SR.PlatformNotSupported_AccessControl</GeneratePlatformNotSupportedAssemblyMessage>
-    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;netcoreapp2.1;$(NetCoreAppCurrent)</TargetFrameworks>
```

Comment:
@ericstj we needed to add the netcoreapp2.1 config in order to workaround the fact that we were missing two declared dependencies in NCA2.1 given that it would take now the NS2.0 assembly as the compile asset. This workaround is less than ideal, but likely preferable to add back the refs to the package, do you agree with that

Reply:
I chatted with him on my 1:1 with regards on this, and he said that he prefers doing this than bringing back the ref assemblies and that seems like the best workaround. 

He also suggested that we could exclude the compile asset from the package and so it would only be used to calculate dependencies but that either way was fine.

Reply:
I'm a little surprised we didn't copy the PackageDependencies from netstandard2.0 to netcoreapp2.1 here: https://github.com/dotnet/arcade/blob/2fd7a99a9bc16b04312b816e3ff0aade830efbeb/src/Microsoft.DotNet.Build.Tasks.Packaging/src/build/Packaging.targets#L680 

Barring that another hack we could use here would be to copy the netstandard2.0 harvested asset into lib/netcoreapp2.1 as well.